server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/ConsistencyTasksIntegrationTest.java

```diff
@@ -443,4 +508,46 @@ void recomputeQuotasShouldSolveNotingWhenNoInconsistencies(GuiceJamesServer serv
                 .asLong())
             .isEqualTo(1);
     }
+
+    @Test
+    void solveMessagesInconsistencyShouldSolveNothingWhenNoInconsistencies(GuiceJamesServer server) throws MailboxException {
+        MailboxProbeImpl probe = server.getProbe(MailboxProbeImpl.class);
+        MailboxPath inbox = MailboxPath.inbox(BOB);
+        MailboxId mailboxId = probe.createMailbox(inbox);
+
+        ComposedMessageId messageId = probe.appendMessage(BOB.asString(), inbox,
+            new ByteArrayInputStream(MESSAGE_CONTENT_AS_BYTES), DATE,
+            !IS_RECENT, FLAGS);
+
+        // schema version 6 or higher required to run solve mailbox inconsistencies task
+        String taskId = with().post(UPGRADE_TO_LATEST_VERSION)
```

Comment:
How about doing this `UPGRADE_TO_LATEST_VERSION` in a @BeforeEach setUp method?

Reply:
but not every test requires it?

Reply:
Then let's move it to the beginning of the test cases requiring it.